packages/aws-cdk/lib/init-templates/app/util/os.ts

```diff
@@ -24,30 +22,26 @@ export async function shell(command: string[], options: ShellOptions = {}): Prom
 
     // Both write to stdout and collect
     child.stdout.on('data', chunk => {
-      if (!options.quiet) {
-        process.stdout.write(chunk);
-      }
+      process.stdout.write(chunk);
       stdout.push(chunk);
     });
 
     child.once('error', reject);
 
     child.once('exit', code => {
       if (code === 0) {
-        resolve(Buffer.concat(stdout).toString('utf-8'));
+        resolve(Buffer.from(stdout).toString('utf-8'));
       } else {
-        reject(new Error(`${renderCommandLine(command)} exited with error code ${code}`));
+        reject(new Error(`${options.errorMessage} exited with error code ${code}`));
```

Comment:
This used to say something like

```
dotnet add bla exited with error code 1
```

And will now say

```
Could not add project project.fsproj to solution project.sln exited with code 1
```

This is hiding information now, we can't tell the command line anymore.

Reply:
I pulled this from what we were actually outputting in the original version of the hook file, which was 
```
reject(new Error(`Could not add project %name.PascalCased%.csproj to solution %name.PascalCased%.sln. Error code: ${code}`));
```
Both pieces of context are probably helpful in this case so I'll combine the two for the error message.